src/rest-client.ts

```diff
@@ -2746,46 +2399,29 @@ export class RestClient extends BaseRestClient {
     return this.getPrivate('/api/v5/users/subaccount/list', params);
   }
 
-  /**
-   * @deprecated Breaking change: this function will change soon to accept a single object for all REST API parameters.
-   * See issue https://github.com/tiagosiebler/okx-api/issues/88 for more details.
-   */
-  resetSubAccountAPIKey(
-    subAcct: string,
-    apiKey: string,
+  resetSubAccountAPIKey(params: {
+    subAcct: string;
+    apiKey: string;
     options?: {
       label?: string;
       perm?: string;
       ip?: string;
-    },
-  ): Promise<SubAccountAPIReset[]> {
-    return this.postPrivate('/api/v5/users/subaccount/modify-apikey', {
-      subAcct,
-      apiKey,
-      ...options,
-    });
+    };
```

Comment:
Same here for "options". That object doesn't actually exist as a parameter for this API call, the contents of "options" need to be a part of "params" (so you can remove options & keep the contents).